operators/pkg/controller/common/annotation/controller_version.go

```diff
@@ -112,8 +111,6 @@ func ReconcileCompatibility(client k8s.Client, obj runtime.Object, selector labe
 
 	// if the current version is gte the minimum version then they are compatible
 	if currentVersion.IsSameOrAfter(*minVersion) {
-		log.V(1).Info("Current controller version on resource is compatible with running controller version", "controller_version", ctrlVersion,
-			"resource_controller_version", currentVersion, "namespace", namespace, "name", name)
 		return true, nil
 	}
 
```

Comment:
Reason for removing those: let's log the exceptional case (incompatible version), not the happy path that happens at all reconciliations.